### PR TITLE
feat(ui-kit:-timeline): 13937-add-configuration-for-timline-paddings

### PR DESCRIPTION
**Pull request by dqunbp**
- Add `margin` prop to `Timeline` component to allow managing timeline items' margin
- Add `prettier` recommended to `eslint` conf

With default margins:
image
With zero margins:
image

### CODE REVIEW

**Comment by IlyaIzr** [comment]
@propakov what was our agreement on updating fixtures files? Is it all good?

**Comment by propakov** [comment]
> @propakov what was our agreement on updating fixtures files? Is it all good?

I don't remember we had such. But what exactly bothers you? @IlyaIzr
It's important to commit cosmos.userdeps.js file, too build cosmos ui right, here it's all good.

File: packages/ui-kit/src/Timeline/implementation/vis-timeline/toVisTimelineOptions.tsx [hunk 1]
```diff
@@ -43,5 +47,9 @@ export const toVisTimelineOptions = (options: TimelineOptions): VisTimelineOptio
     };
   }
 
+  if (typeof options.margin !== 'undefined') {
```
Comment (propakov) [inline packages/ui-kit/src/Timeline/implementation/vis-timeline/toVisTimelineOptions.tsx:50]: could be just `options.margin !== undefined`

**Review by propakov — APPROVED** [review]
Good job! 👍 

BTW i've found that out command "lint:fix": "eslint \"*/**/*.{ts,tsx}\" --quiet --fix",
doesn't apply autofix because of --quiet (it's a lot of files where prettier need to be fixed)
Not sure in this PR or not, but would be cool to fix them all.

File: packages/ui-kit/src/Timeline/implementation/vis-timeline/toVisTimelineOptions.tsx (hunk 1 above)
Comment (dqunbp) [reply packages/ui-kit/src/Timeline/implementation/vis-timeline/toVisTimelineOptions.tsx:50]: @propakov Yes we can, `typeof` approach is a more universal and recommended way to check if variables are `undefined`.

**Comment by dqunbp** [comment]
@propakov Looks like lint-staged scripts do not run when I commit some changes. I will take a look

**Pull request merged**